app.js

```
@@ -1,88 +1,93 @@
+});
+
+// Export app for testing
 module.exports = app;
+
+// Helper Functions
+
+// Handle Method Override
+function handleMethodOverride(req, res) {
+  if (req.body && typeof req.body === 'object' && '_method' in req.body) {
+    const method = req.body._method;
+    delete req.body._method;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)